en/docs/setup/running-the-product.md

```diff
@@ -115,31 +102,28 @@ the Command Prompt as described below.
     `          dtterm         ` at the Prompt, and then press **Enter**
    , to open a Command Prompt.
 2.  Execute the following command:
-    `          <PRODUCT_HOME>/bin/          bash          wso2server.sh.         `
+    `          <IS_HOME>/bin/          bash          wso2server.sh.         `
 3.  The operation log appears in the command window. When the product
     server has successfully started, the log displays the message "WSO2
     Carbon started in 'n' seconds".
 
-You need to do the following modification to the
-`          <PRODUCT_HOME>/bin/wso2server.sh         ` file, to start the
-product as a service/nohup mode in Solaris.
-
-1.  Open the `            <PRODUCT_HOME>/bin/wso2server.sh           `
-    file in a text editor.
-2.  Search for the following occurrences: **nohup sh
-    "$CARBON\_HOME"/bin/wso2server.sh $args \> /dev/null 2\>&1 &**
-3.  Replace those occurrences with the following: **nohup <span
-    class="underline">bash</span> "$CARBON\_HOME"/bin/wso2server.sh
-    $args \> /dev/null 2\>&1 &**
-
-    !!! tip
-    
-        The only change is replacing `             sh            ` with
-        `             bash            ` . This is required only for Solaris.
-    
-
-4.  Start the product by following the [above
-    instructions](#RunningtheProduct-instructions).
+!!! info 
+    You need to do the following modification to the
+    `          <IS_HOME>/bin/wso2server.sh         ` file, to start the
+    product as a service/nohup mode in Solaris.
+
+    1.  Open the `            <IS_HOME>/bin/wso2server.sh           `
+        file in a text editor.
+    2.  Search for the following occurrences: **nohup sh
+        "$CARBON\_HOME"/bin/wso2server.sh $args \> /dev/null 2\>&1 &**
+    3.  Replace those occurrences with the following: **nohup <span
+        class="underline">bash</span> "$CARBON\_HOME"/bin/wso2server.sh
+        $args \> /dev/null 2\>&1 &**
+        
+        !!! tip
+            The only change is replacing `             sh            ` with
+            `             bash            ` . This is required only for Solaris.
+    4.  Start the product by following the [above instructions](#running-the-product).
```

Comment:
since this link is not an anchor link and is just linking to the same page, maybe we can remove the link and just have the text "above instructions"

Reply:
modified.